components/style/inline.rs

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use computed_values::white_space;
+
+pub trait WhitespaceMethods {
```

Comment:
If I read your comment in #8128 right,  you said that these methods should live in an extension trait in a file called `inline.rs`. Do you mean that I should disregard that comment and implement these methods in `properties.mako.rs` instead?